Gemfile

```diff
@@ -108,7 +111,7 @@ group :development do
   gem "web-console", "~> 3.0"
 
   # Reenable after https://github.com/rails/rails/issues/26158 is fixed
-  # gem 'listen', '~> 3.0.5'
```

Comment:
http://guides.rubyonrails.org/security.html#custom-credentials doesnt work without this gem